test/e2e/suites/e2e/e2e_suite_test.go

```diff
@@ -52,7 +52,7 @@ func TestE2E(t *testing.T) {
 
 var _ = SynchronizedBeforeSuite(func() []byte {
 	data := shared.Node1BeforeSuite(e2eCtx)
-	createTestVolumeType(e2eCtx)
+	getVolumeType(e2eCtx)
```

Comment:
Is this a sanity check or do we actually need it?
I would prefer to get rid of it

Reply:
As I understand it, we need this if you're considering using a different volume type than our default `test-volume-type`. However, my tests show that removing the function doesn't cause issues. But, do we want the flexibility to specify a different volume type in the config file?

Reply:
You mean `createTestVolumeType`? I'm wondering what the point is with `getVolumeType`? My understanding is that it just checks that the volume is there but I may have missed something. Does it get the volume type ID so that can be used later?

Reply:
You are right, `getVolumeType` was only for confirming the volume type.